akka-actor/src/main/scala/akka/event/Logging.scala

```diff
@@ -816,6 +816,11 @@ object Logging {
   /** INTERNAL API, Marker interface for LogEvents containing Markers, which can be set for example on an slf4j logger */
   sealed trait LogEventWithMarker extends LogEvent {
     def marker: LogMarker
+    /** Appends the marker to the Debug/Info/Warning/Error toString representations */
+    override def toString = {
+      val s = super.toString
+      s.substring(0, s.length - 1) + "," + marker + ")"
```

Comment:
ok, that is good, but `toString` is not used for printing the actual log message.
I was thinking it should be printed something like
```
[WARN] [SECURITY] ...
```